### PR TITLE
Use OBS containers

### PR DESCRIPTION
**Pull request by juliogonzalez**
## What does this PR change?

Changes all references to `registry.mgr.suse.de` to `registry.opensuse.org` and the right names, so we use the containers from OBS.

Question: what should we do about...

```
$ grep -ir 'registry.mgr' *
testing/salt/run_salt_tests.sh:# docker run -t -i --privileged --device /dev/mem --rm -v "/path/to/git:/manager" registry.mgr.suse.de/suma-head-salt /bin/bash
```
Are we using it? Or can we just remove it? @meaksh?

## GUI diff

No difference.

- [x] **DONE**

## Documentation

I modified the `README.md` file

- [x] **DONE**

## Test coverage

This is changing tests

- [x] **DONE**

## Links

Related: https://github.com/SUSE/spacewalk/issues/14921

- [x] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

**Review by mcalmer — CHANGES_REQUESTED** [review]
Missing spacewalk-backend: backend/Makefile.backend

and grep for registry.mgr.suse.de . There seems to be some more.

**Comment by juliogonzalez** [comment]
> Missing spacewalk-backend: backend/Makefile.backend

I hate this. We should have the tests in one place :\

File: backend/server/test/unit-test/Makefile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Docker tests variables
 DOCKER_CONTAINER_BASE = manager_python_testing
 DOCKER_TESTED_BRANCH  = head
-DOCKER_REGISTRY       = registry.mgr.suse.de
+DOCKER_REGISTRY       = registry.opensuse.org
```
Comment (juliogonzalez) [inline backend/server/test/unit-test/Makefile:4]: I wonder if we should not just remove this file.

It says below that this is using an Oracle container?

Comment (mcalmer) [reply backend/server/test/unit-test/Makefile:4]: I think this is unused. AFAIK this is called from backend/test/docker-backend-server-tests.sh

Comment (juliogonzalez) [reply backend/server/test/unit-test/Makefile:4]: I could change `backend/test/docker-backend-server-tests.sh`, but then that means no tests for the backend anymore :-?

@meaksh?

Something tell me that I need to rollback this change for `backend/server/test/unit-test/Makefile` until we can clarify.

**Comment by juliogonzalez** [comment]
> Found another file which needs changes. /java/Makefile.docker should also specify the images from the external registry

Hopefully everything is done now:
```
juliogonzalez@juliogonzalez:~/Documents/development/git/suse/spacewalk (public-registry=)$ grep -ir 'registry.suse.de' *
juliogonzalez@juliogonzalez:~/Documents/development/git/suse/spacewalk (public-registry=)$ 
```

**Review by mcalmer — APPROVED** [review]
Let's try it :-)

File: backend/server/test/unit-test/Makefile (hunk 1 above)
Comment (juliogonzalez) [reply backend/server/test/unit-test/Makefile]: I talked to Pablo. We are removing this file, and created https://github.com/SUSE/spacewalk/issues/15099

**Comment by juliogonzalez** [comment]
I will merge when the branch is not locked.

**Pull request merged**